addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -5,7 +5,15 @@
             <t t-if="messageInReplyToView">
                 <t t-if="!messageInReplyToView.messageView.message.parentMessage.isEmpty">
                     <b class="o_MessageInReplyToView_author text-muted ml-2">@<t t-esc="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
-                    <span class="o_MessageInReplyToView_body ml-1" t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody" t-on-click="messageInReplyToView.onClickReply"/>
+                    <span class="o_MessageInReplyToView_body ml-1" t-on-click="messageInReplyToView.onClickReply">
+                        <t t-if="messageInReplyToView.hasBodyBackLink">
+                            <t t-raw="messageInReplyToView.messageView.message.parentMessage.prettyBody"/>
+                        </t>
+                        <t t-if="messageInReplyToView.hasAttachmentBackLink">
+                            <span class="font-italic mr-2">Click to see the attachments</span>
```

Comment:
As this is v15 the new string should be added to pot file. I'll do it to merge it now.